lib/doc_builder_testing/web_doc_builder_wrapper.rb

```diff
@@ -24,6 +24,11 @@ def initialize(**args)
     @payload_params = { payload: {} }
   end
 
+  # @return [String] Url of DocBuilder server
+  def server_url
```

Comment:
Can be `attr_reader :uri` and access like `kim.uri`

Reply:
Yes, this way is definitely better. Thank you!